conf/messages.cy

```diff
@@ -367,7 +367,7 @@ cds.company.name.label=Enw’r cwmni
 cds.partnership.name.label=Enw’r bartneriaeth gofrestredig
 cds.nino.label=Rhif Yswiriant Gwladol
 cds.utr.label=Rhif Cyfeirnod Unigryw y Trethdalwr (UTR)
-cds.company.utr.label=Rhif Cyfeirnod Unigryw y Trethdalwr (UTR) ar gyfer Treth Gorfforaeth
+cds.company.utr.label=TRANSLATE Corporation Tax UTR
```

Comment:
Is the welsh translation not available yet?

Reply:
There is a change in the UTR label, earlier we had 'Corporation Tax UTR **number**' as suffix, but this has been removed and will be 'Corporation Tax UTR' now. Hence referred as to be TRANSLATE in future for Welsh related labels.